drivers/virtualbox/virtualbox.go

```diff
@@ -174,23 +179,71 @@ func (d *Driver) SetConfigFromFlags(flags drivers.DriverOptions) error {
 
 func (d *Driver) PreCreateCheck() error {
 	// Check that VBoxManage exists and works
-	if err := vbm(); err != nil {
-		return err
+	return vbm()
+}
+
+// IsVTXDisabled checks if VT-X is disabled in the bios. If it is, the vm will fail to start.
+// If we can't be sure it is disabled, we carry on and will check the vm logs after it's started.
+func (d *Driver) IsVTXDisabled() bool {
+	// TODO: Check on linux/OSX
```

Comment:
@hairyhenderson the code doesn't compile on linux. I'll put that on hold. Windows was really the priority.